src/core/lv_obj_style.c

```diff
@@ -286,6 +298,9 @@ void lv_obj_refresh_style(lv_obj_t * obj, lv_style_selector_t selector, lv_style
     bool is_ext_draw = lv_style_prop_has_flag(prop, LV_STYLE_PROP_FLAG_EXT_DRAW_UPDATE);
     bool is_inheritable = lv_style_prop_has_flag(prop, LV_STYLE_PROP_FLAG_INHERITABLE);
     bool is_layer_refr = lv_style_prop_has_flag(prop, LV_STYLE_PROP_FLAG_LAYER_UPDATE);
+    bool is_transform_rotation = (prop == LV_STYLE_TRANSFORM_ROTATION) ? true : false;
```

Comment:
```suggestion
    bool is_transform_rotation = (prop == LV_STYLE_PROP_ANY || prop == LV_STYLE_TRANSFORM_ROTATION) ? true : false;
```
We need t handle `LV_STYLE_PROP_ANY` but in this case `transform_changed` will be always incremented.

Reply:
this may be a basic question, 
could you let me know why we handle LV_STYLE_PROP_ANY?

Reply:
`LV_STYLE_PROP_ANY` means we don't know the property. E.g. because a style was added with multiple proeprties.